### ISSUE TITLE
Add implementation of EventStore integrated with EventStoreDB

### ISSUE THREAD

**Issue by nowakprojects** [johanhaleby/occurrent#59]
Consider which EventStoreDB client should be used.

An Official Java client is under development: 
https://github.com/EventStore/EventStoreDB-Client-Java

**Comment by johanhaleby** [comment]
Yes, I've been interested in this as well, but last time I looked there were no Java binaries available for the Java Client. 

But a second point is that I'm not sure how much value it would add besides that it would be a nice experiment. Occurrent would probably just be a mapping layer from CloudEvent's to the EventStoreDB data model. But maybe that would be good enough? It _should_ (hopefully) be relatively straightforward to do (but one never knows until you've tried :)).
